docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+* validated by a streamlet (splitter) that separates valid and invalid records
+* logged in to loggers to be checked at output (egress)
+
+Each of the above terminologies (ingress, processor, splitter and egress) are explained in the [Concepts Guide](https://blank.com). Here's an overview of the application pipeline architecture:
+
+![Application Pipeline](images/pipe.001.jpeg?raw=true "Application Pipeline")
+
+One of the important features of cloudflow architecture is the complete separation of the components from how they are connected as part of the pipeline. The _streamlets_ described above are the individual building blocks of the pipeline. You can connect them using a declarative language that forms the _blueprint_ of the pipeline. Streamlets can be shared across blueprints making them individual reusable objects. And just like streamlets, blueprints also form an independent component of a cloudflow application.
+
+### Project structure
+
+Here's how we would structure a typical cloudflow application as a project.
+
+```
+   |-project
+   |-src
+   |---main
+   |-----avro
+   |-----blueprint
+   |-----resources
+   |-----scala
+   |-------sensordata
+   |-build.sbt
+```
+
+This is a Scala project and we have the following structural components at the leaf level of the above tree:
+
+* **avro** : contains the avro schema of the domain objects
+* **blueprint** : contains the blueprint of the application in a file named `blueprint.conf`
+* **scala** : contains the source code of the application under the package name `sensordata`
+* **build.sbt** : the sbt build script
+
+### The sbt build script
+
+Here's a minimal version of the sbt build script:
+
+```
+import sbt._
+import sbt.Keys._
+
+lazy val sensorData =  (project in file("."))
+  .enablePlugins(CloudflowAkkaStreamsApplicationPlugin)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.lightbend.akka"     %% "akka-stream-alpakka-file"  % "1.1.2",
+      "com.typesafe.akka"      %% "akka-http-spray-json"      % "10.1.10",
+      "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
+      "com.typesafe.akka"      %% "akka-http-testkit"         % "10.1.10" % "test"
+    ),
+    name := "sensor-data",
+    organization := "com.lightbend",
+
+    scalaVersion := "2.12.10",
+    crossScalaVersions := Vector(scalaVersion.value)
+  )
+```
+
+cloudflow offers a few plugins that abstract quite a bit of boilerplates necessary to build a complete application. In this example we use the plugin `CloudflowAkkaStreamsApplicationPlugin` that provides you all building blocks of developing an Akka Streams based cloudflow application.
+
+> **Note:** You can use multiple plugins to develop an application that uses multiple runtimes (Akka, Spark, Flink etc.). For simplicity of this example we will be using only one.
+ 
+The above build script is standard Scala sbt - the only difference is the lugin which we provide as part of cloudflow.
```

Comment:
s/lugin/plugin

Reply:
Could you elaborate a bit more about the sbt plugins here and what they do. Maybe a diagram could help.

Reply:
@skonto Intentionally I avoided the details. The purpose of this doc is that the user will follow it and have the application (described here) up and running. I guess we will have more details on the sbt plugins in developer docs. Once we have them we can refer to that from here.